### PR TITLE
Adding clarifications to HS2 walkthrough/faq and improving sample XML.

### PR DESCRIPTION
**Pull request by rlee-sap**

### CODE REVIEW

**Review by stuart-ng-van — APPROVED** [review]

File: src/api-guides/travel/hotelservice2/faq.markdown [hunk 1]
```diff
@@ -93,6 +99,20 @@ A: This value indicates whether the guest is a child or adult. SAP Concur only s
 
 ***
 
+Q: What comprises the content of a rate description in the rates listing?
+
+A: The rate description is made up of the RateDescription, RatePlanDescription, RoomDescription, and MealsIncluded (breakfast) from the Availability response.
```
Comment (MarcBell-SAP) [inline src/api-guides/travel/hotelservice2/faq.markdown]:
```suggestion
A: The rate description is made up of the `RateDescription`, `RatePlanDescription`, `RoomDescription`, and `MealsIncluded` (breakfast) from the Availability response.
```

File: src/api-guides/travel/hotelservice2/faq.markdown [hunk 2]
```diff
@@ -93,6 +99,20 @@ A: This value indicates whether the guest is a child or adult. SAP Concur only s
...
+
+***
+
+Q: What content is used to populate the Rules and Cancellation Policy?
+
+A: The text is generated using the content from Availability (or RateDetails when used). Including the following:
```
Comment (MarcBell-SAP) [inline src/api-guides/travel/hotelservice2/faq.markdown]:
```suggestion
A: The text is generated using the content from Availability (or `RateDetails` when used). Including the following:
```

File: src/api-guides/travel/hotelservice2/hotel-shop-book-cancel.markdown [hunk 3]
```diff
@@ -39,15 +43,17 @@ Basic scenario describing functionality provided by the Hotel Service v2 API inc
...
+    
     [Sample search request](./sample-requests/general-walkthrough/search-rq.xml)
 
     [Sample search response](./sample-requests/general-walkthrough/search-rs.xml)
 
-    Out of 100 hotels returned in the response from the Hotel Supplier, the first 10 hotels are the Most Preferred hotels within a 30km radius. The next 10 hotels are the Preferred hotels within a 30km radius. The remaining 80 hotels are hotels with no preference within a 5km radius. Note: The preference level is defined by the HotelPreference element in the TPA_Extensions, which is outlined in Search.
+    Out of 100 returned hotels in the response from the Hotel Supplier, the response may contain 10 Most Preferred hotels within the 30 mile radius, 10 Preferred within the 30 mile radius, and 10 Less Preferred hotels within the 30 mile radius. The remaining 70 hotels would be non-preferred hotels within the 5 mile radius. Note: The preference level is defined by the HotelPreference element in the TPA_Extensions, which is outlined in Search.
```
Comment (MarcBell-SAP) [inline src/api-guides/travel/hotelservice2/hotel-shop-book-cancel.markdown]:
```suggestion
    Out of 100 returned hotels in the response from the Hotel Supplier, the response may contain the 10 `most_preferred` hotels within a 30 mile radius, the 10 `preferred` hotels within a 30 mile radius, and the 10 `less_preferred` hotels within a 30 mile radius. The remaining 70 hotels would be `not_preferred` hotels within a 5 mile radius. Note: The preference level is defined by the `HotelPreference` element in the TPA_Extensions, which is outlined in Search.
```

File: src/api-guides/travel/hotelservice2/hotel-shop-book-cancel.markdown [hunk 4]
```diff
@@ -103,24 +108,23 @@ Basic scenario describing functionality provided by the Hotel Service v2 API inc
 
     [Sample descriptive info response](./sample-requests/general-walkthrough/hoteldetail-rs.xml)
 
-    Please refer to the HotelDescriptiveInfo [FAQ](./faq.html#hotel-descriptive-info) for additional information.
-    
+    Please refer to the [FAQ](./faq.html#hotel-descriptive-info) for additional information regarding HotelDescriptiveInfo.
```
Comment (MarcBell-SAP) [inline src/api-guides/travel/hotelservice2/hotel-shop-book-cancel.markdown]:
```suggestion
    Please refer to the [FAQ](./faq.html#hotel-descriptive-info) for additional information regarding `HotelDescriptiveInfo`.
```

File: src/api-guides/travel/hotelservice2/hotel-shop-book-cancel.markdown [hunk 5]
```diff
@@ -103,24 +108,23 @@ Basic scenario describing functionality provided by the Hotel Service v2 API inc
...
+7. Note each rate displays a Rules and cancellation policy link. When the business traveler clicks the link, if the cancellation details were deferred as indicated by [TPA_Extensions/RateDetailsInd](https://developer.concur.com/api-reference/direct-connects/hotel-service-2/Availability.html#res-schema), a [RateDetails](https://developer.concur.com/api-reference/direct-connects/hotel-service-2/Rate-details.html) request will be invoked. The response content will be displayed in a pop-up window.
 
     <a href='./images/general-walkthrough/rules-and-cancellation-policy.png'><img style="max-width:300px" src="./images/general-walkthrough/rules-and-cancellation-policy.png"/></a>
 
     [Sample rate details request](./sample-requests/general-walkthrough/ratedetails-rq.xml)
 
     [Sample rate details response](./sample-requests/general-walkthrough/ratedetails-rs.xml)
 
-    Please refer to the RateDetails [FAQ](./faq.html#availability) for additional information.
+    Please refer to the [FAQ](./faq.html#availability) for additional information regarding RateDetails.
```
Comment (MarcBell-SAP) [inline src/api-guides/travel/hotelservice2/hotel-shop-book-cancel.markdown]:
```suggestion
    Please refer to the [FAQ](./faq.html#availability) for additional information regarding `RateDetails`.
```

File: src/api-guides/travel/hotelservice2/hotel-shop-book-cancel.markdown [hunk 6]
```diff
@@ -136,28 +140,37 @@ Basic scenario describing functionality provided by the Hotel Service v2 API inc
...
+11. When the reservation details are displayed SAP Concur will invoke a [ReadItinerary](https://developer.concur.com/api-reference/direct-connects/hotel-service-2/Read-Itinerary.html) request. If a Read request does not arrive within 5 minutes for a given Reservation, then the supplier should treat that Reservation as an orphan and should thus seek to cancel it. The business traveler completes their reservation by confirming the booking.
 
     [Sample read request](./sample-requests/general-walkthrough/read-rq-load-trip.xml)
 
     [Sample read response](./sample-requests/general-walkthrough/read-rs-load-trip.xml)
 
-    Please refer to the ReadItinerary [FAQ](./faq.html#read-itinerary) for additional information.
+    Please refer to the [FAQ](./faq.html#read-itinerary) for additional information regarding ReadItinerary.
```
Comment (MarcBell-SAP) [inline src/api-guides/travel/hotelservice2/hotel-shop-book-cancel.markdown]:
```suggestion
    Please refer to the [FAQ](./faq.html#read-itinerary) for additional information regarding `ReadItinerary`.
```

**Review by MarcBell-SAP — CHANGES_REQUESTED** [review]
@rlee-sap Review and approve. Then I can merge it.

**Comment by MarcBell-SAP** [comment]
:shipit:

**Review by MarcBell-SAP — APPROVED** [review]

**Pull request merged**